embassy-time-driver/src/lib.rs

```diff
@@ -70,6 +70,23 @@ mod tick;
 /// This value is specified by the [`tick-*` Cargo features](crate#tick-rate)
 pub const TICK_HZ: u64 = tick::TICK_HZ;
 
+const fn gcd(a: u64, b: u64) -> u64 {
```

Comment:
please just copypaste this instead of adding more public API to `embassy-time-driver`.

I would prefer to keep the `embassy-time-driver` public API surface ruthlessly barebones, to reduce the chances it needs a major version bump.

Reply:
I see, that's understandable. Fixed it to avoid changes in time-driver.